### DISCUSSION TITLE
v1.14.0

### DISCUSSION THREAD

**Discussion by Kovah** [Kovah/LinkAce#746] [Announcements]
**Support for PHP 8.0 was dropped. LinkAce only supports PHP 8.1, 8.2 and 8.3.**

* Update for the Romanian translation.
* Small Interface fixes.
* Large-scale update of used dependencies.
* Base Node version used in the project is not v20 LTS.

<hr /><em>This discussion was created from the release <a href='https://github.com/Kovah/LinkAce/releases/tag/v1.14.0'>v1.14.0</a>.</em>

**Comment by chrissawyerfan4** [comment]
Not sure if you get a ping from this comment, but the [docs](https://www.linkace.org/docs/v1/setup/setup-without-docker/) still say 8.0 is supported and the [developer docs](https://github.com/Kovah/LinkAce/blob/cd107987c3379d9d04c18b9798bca46d4609a973/CONTRIBUTING.md) still mention 7.4.

Given how often this changes, I wonder if there's a way to centrally encode this somewhere, or whether saying "a [currently-supported PHP version](https://www.php.net/supported-versions.php)" may be the easier strategy since that seems to be the philosophy (which, by the way, I find a bit hard to follow since my Linux distribution supports them way longer and there are barely any untrusted inputs on a single-user LinkAce setup; but that nuance is perhaps beyond the scope of a broad, all-purposes recommendation).

**Reply by Kovah** [reply]
Yeah I got the comment. 😬 

Thanks for the heads-up on the incorrect documentation. Will have a look at this and update it soon to match the current supported versions. Technically, it's already written down in the [SECURITY.md file](https://github.com/Kovah/LinkAce/blob/main/SECURITY.md), but I guess it would be easier to have it only in one place and reference it from anywhere else. 🤔 

Definitely understand that the supported PHP version is a bit strict, as LinkAce is not really a public app and some systems support older versions way longer. I've chosen this strategy to keep it as simple as possible. If there's a specific reason to also support older versions, I will reconsider this. Right now, I don't see any specific reason for that.
